src/components/forms/NumberField/NumberField.stories.mdx

```diff
@@ -26,7 +27,14 @@ const generalTest = async canvasElement => {
 <Meta
   title="Pure React Components / Forms / NumberField"
   component={NumberField}
+  decorators={[FormikDecorator]}
   parameters={{
+    docs: {
+      source: {
+        type: 'dynamic',
+        excludeDecorators: true,
+      },
+    },
```

Comment:
This leads to useful code previews in the "Docs" tab:

![image](https://github.com/open-formulieren/open-forms-sdk/assets/5518550/b5b17376-acc9-4001-bd80-27100c219b4d)